package.json

```diff
@@ -47,9 +47,13 @@
     "lint:js": "eslint .",
     "preinstall": "node tasks/checkNodeVersion.js",
     "postinstall": "npm run python",
-    "python": "virtualenv .python && run-script-os",
-    "python:darwin:linux": "PATH=.python/bin:${PATH} pip install -r requirements.txt",
-    "python:win32": "bash -c 'PATH=.python/Scripts:${PATH} pip install -r requirements.txt'",
+    "python": "npm run python:start",
+    "python:start": "run-script-os",
+    "python:start:darwin:linux": "python2 -m pip install virtualenv && python2 -m virtualenv .python && npm run python:end",
+    "python::start:win32": "virtualenv .python && npm run python:end",
```

Comment:
`python:start:win32` instead of `python::start:win32` here

Reply:
whoops 👍